### DISCUSSION TITLE
Odd Issue on firestick

### DISCUSSION THREAD

**Discussion by kiasho** [awawa-dev/HyperHDR#655] [General]
I am having an issue with streaming apps on firestick the hyperhdr led's work with netflix and amazon but I'm getting no led's when using disney+ or apple tv and I'm not sure why.

To add to this i checked both apps through my sky q box and the led's work on both apps as long as i dont have the sky q box set to uhd 10bit they only work with the box in 1080p.

**Comment by awawa-dev** [comment]
> I am having an issue with streaming apps on firestick the hyperhdr led's work with netflix and amazon but I'm getting no led's when using disney+ or apple tv and I'm not sure why.

HDCP (or using not-supported resolution or format like Dolby Vision: you must check your settings)
